src/momento/cache_client.py

```diff
@@ -9,35 +9,6 @@
 from momento.config import Configuration
 from momento.errors import UnknownException
 from momento.requests import CollectionTtl, SortOrder
-from momento.responses.data.sorted_set.fetch import CacheSortedSetFetchResponse
-from momento.responses.data.sorted_set.get_rank import CacheSortedSetGetRankResponse
-from momento.responses.data.sorted_set.get_score import (
-    CacheSortedSetGetScore,
-    CacheSortedSetGetScoreResponse,
-)
-from momento.responses.data.sorted_set.get_scores import (
-    CacheSortedSetGetScores,
-    CacheSortedSetGetScoresResponse,
-)
-from momento.responses.data.sorted_set.increment_score import (
-    CacheSortedSetIncrementScoreResponse,
-)
-from momento.responses.data.sorted_set.put_element import (
-    CacheSortedSetPutElement,
-    CacheSortedSetPutElementResponse,
-)
-from momento.responses.data.sorted_set.put_elements import (
-    CacheSortedSetPutElements,
-    CacheSortedSetPutElementsResponse,
-)
-from momento.responses.data.sorted_set.remove_element import (
-    CacheSortedSetRemoveElement,
-    CacheSortedSetRemoveElementResponse,
-)
-from momento.responses.data.sorted_set.remove_elements import (
-    CacheSortedSetRemoveElements,
-    CacheSortedSetRemoveElementsResponse,
-)
```

Comment:
what's going on with these changes?

Reply:
These hadn't been added to `momento.responses` for some reason, hence the really long `from momento.responses.data....`. While I was here I added those to `momento.responses` and moved these import lines below.

Reply:
:ack: